### PR TITLE
(SERVER-809) Launch sbt from script

### PR DESCRIPTION
**Pull request by jpinsonault**
ping @nwolfe

### CODE REVIEW

File: simulation-runner/run_sbt.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```
Comment (nwolfe) [inline simulation-runner/run_sbt.sh]: Man, so many similar files.

I think we should get rid of this one and just inline the call to `sbt run` in the `on(gatling, ...)` call in `20_run_sbt.rb`.

Comment (jpinsonault) [inline simulation-runner/run_sbt.sh]: Missed this comment, sorry. Yeah that sounds reasonable to me, I don't have a strong opinion

**Comment by jpinsonault** [comment]
@nwolfe Inlined the sbt command

**Comment by nwolfe** [comment]
:+1:

**Pull request merged**